src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -69,7 +69,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     // Get the input and time from the form.
     String textEntryString = request.getParameter("text");
     String moodScaleString = request.getParameter("mood");
-    String songEntryString = request.getParameter("song");
```

Comment:
Could you add this back to the code, I see it is later referenced when storing the song to Datastore via the journalEntity?

Reply:
I think I messed up when I did merged in master